core/primitives/base58_codec.cpp

```diff
@@ -54,9 +58,9 @@ namespace kagome::primitives {
     int zeroes = 0;
     int length = 0;
     while (str[0] == '1') {
+      str = str.substr(1);
```

Comment:
Why? It worse way because it makes data copy for each iteration.

Reply:
it's stringview